src/test/regress/expected/connectivity_checks.out

```diff
@@ -0,0 +1,6 @@
+SELECT count(*) FROM citus_check_cluster_node_health() WHERE success IS NOT true;
```

Comment:
I wonder if our *official* suggestion for querying the cluster health should be:
```SQL
-- returns sourceHost:sourcePort targetHost:targetPort success
-- success -> true -> connection attempt from source to target succeeded
-- success -> false -> connection attempt from source to target failed
-- success -> NULL -> connection attempt from the current node to target failed 
select bool_and(COALESCE(success, false)) from citus_check_cluster_node_health();
```

Whenever this query return false, there is a connectivity issue, check in detail.

Reply:
also, maybe we should rename `success ` -> `result` ?